domain/src/main/java/org/oppia/android/domain/classify/rules/numericinput/NumericInputIsGreaterThanRuleClassifierProvider.kt

```diff
@@ -11,7 +11,8 @@ import javax.inject.Inject
  *
  * https://github.com/oppia/oppia/blob/37285a/extensions/interactions/NumericInput/directives/numeric-input-rules.service.ts#L27
  */
-internal class NumericInputIsGreaterThanRuleClassifierProvider @Inject constructor(
+// TODO(#1580): Re-restrict access using Bazel visibilities
```

Comment:
adding this a reference to check before merge if all the files have this todo because till the time it merge, new files are merging from other PR's.

Reply:
I checked this before I sent the PR back for review, but I'll double check before merging.